bin/repl.js

```diff
@@ -245,16 +245,16 @@ async function evaluate (cmd, ctx, file, callback) {
   if (!isTry && !/import\s*\(/.test(cmd)) {
     if (/^\s*await/.test(cmd)) {
       cmd = cmd.replace(/^\s*await\s*/, '')
-      cmd = `void Promise.resolve(${cmd}).then((result) => console.log(io.util.format(result)))`
+      cmd = `void Promise.resolve(${cmd}).then((result) => console.log(socket.util.format(result)))`
     } else if (lastName) {
-      cmd = `${cmd}; io.util.format(${lastName});`
+      cmd = `${cmd}; socket.util.format(${lastName});`
```

Comment:
since it's a proper branded noun, and to make it stand out, might we consider `Socket` instead of `socket`?

Reply:
@jwerle won't the current change break a REPL?

Reply:
why would it break the repl?